src/main/java/io/pivotal/cloudcache/app/CloudcachePizzaStoreApplication.java

```diff
@@ -35,12 +33,12 @@
  *
  */
 @SpringBootApplication
-@EnableDurableClient(id = "pizza-store")
-@EnableClusterAware
-@EnableEntityDefinedRegions
+//@EnableClusterAware
```

Comment:
Is this intentional to be removed? If so, can we delete this line?

Reply:
Another change from original work to make the app work with service gateways. From my own testing, `@EnableClusterAware` is necessary to run the app locally without a cluster. `README.md` has instructions uncomment this annotation to `Try the App in a Local Environment`.